cmd/server-main.go

```diff
@@ -170,7 +169,7 @@ func serverMain(ctx *cli.Context) {
 		fatalIf(errInvalidArgument, "No certificates found for HTTPS endpoints (%s)", globalEndpoints)
 	}
 
-	if !quietFlag {
+	if !quietMode {
```

Comment:
@balamurugana Check for new minio executable and display of text should not happen in json mode